rasa/cli/arguments/x.py

```diff
@@ -12,6 +12,7 @@ def set_x_arguments(parser: argparse.ArgumentParser):
         parser, default=DEFAULT_DATA_PATH, data_type="stories and Rasa NLU "
     )
     default_arguments.add_config_param(parser)
+    default_arguments.add_domain_param(parser)
```

Comment:
The description of domain parameter says "This can be a single YAML file, or a directory that contains several files with domain specifications in it". I am not sure if it's correct because it seems like the rest of the code expects `domain` to be a file, not a directory.

Reply:
Could you please point me to an example? we've changed the description of the "--domain" as of `2.0`, and I was supposed to update to docs as well 😬

Reply:
@degiz here it is: https://github.com/RasaHQ/rasa/blob/master/rasa/cli/arguments/default_arguments.py#L60

Reply:
@alwx right, this doc is something I'm aware of, I meant this:
```
the rest of the code expects domain to be a file, not a directory.
```
I think the rest of the code works correctly with the situation when the domain is a folder.

Reply:
Yes, you seem to be right. Ok, I think we can merge this one then.